### PR TITLE
RPM: adapt ceph-resource-agents package to systemd

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/14828

### CODE REVIEW

File: ceph.spec.in [hunk 1]
```diff
@@ -1175,12 +1175,27 @@ fi
...
 #################################################################################
 %if %{with ocf}
+
 %files resource-agents
 %defattr(0755,root,root,-)
-%dir /usr/lib/ocf
-%dir /usr/lib/ocf/resource.d
-%dir /usr/lib/ocf/resource.d/ceph
-/usr/lib/ocf/resource.d/%{name}/*
+# N.B. %{_libexecdir} is set by unconditional %global to %{_exec_prefix}/lib
```
Comment (ktdreyer) [inline ceph.spec.in]: Does `make install` install these files into the directory specified by `./configure --libdir`, or the one specified by `./configure --libexecdir` ?

Comment (smithfarm) [inline ceph.spec.in]: Good question. It uses the one specified by `./configure --prefix` like so: https://github.com/ceph/ceph/blob/master/src/ocf/Makefile.am#L4-L7

I will replace `%{_libexecdir}` with `%{_prefix}/lib` so it matches.

**Comment by smithfarm** [comment]
Rebased and pushed to gitbuilders.

**Comment by smithfarm** [comment]
Passes the RPM gitbuilders, which is not surprising since `%{with ocf}` evaluates to false.

**Comment by smithfarm** [comment]
Tested in OBS and openSUSE Tumbleweed. All green:

```
vanguard1:~ # rpm -ql ceph-resource-agents
/usr/lib/ocf
/usr/lib/ocf/resource.d
/usr/lib/ocf/resource.d/ceph
/usr/lib/ocf/resource.d/ceph/rbd
```

**Comment by smithfarm** [comment]
Rebased against master: #7894

**Pull request closed without merging**